plane/schema/migrations/20240318145858_support-subdomains.sql

```diff
@@ -0,0 +1,4 @@
+alter table backend add column subdomain varchar(255);
```

Comment:
Since #680 also included a migration, this file will need to be renamed to come after it. Changing the filename to a current timestamp (really any timestamp after `20240319133519`) will do the trick.

Reply:
Thanks for the heads up!

Reply:
huh just rebased, ran all the tests, everything seems fine without changing this... am I missing something or does that mean we're good as is?

Reply:
Tests/running locally should work because they start with a new database and run the migrations in order. I'm worried about applying to a production database where we are adding a migration to an existing stack out of order. I misspoke when I said it _needs_ to be in order (it used to be the case but it's [been fixed](https://github.com/launchbadge/sqlx/issues/869)), but I still think it's good if we try to keep migrations in the migrations folder in the same order that they are actually run in production, in order to ensure that the order of migrations in the code matches the order they were run against prod databases.